### PR TITLE
Create abuse_dropbox_unsolicited_reply-to.yml

### PR DESCRIPTION
**Pull request by zoomequipd**
# Description

Add coverage for abused DropBox

# Associated samples

- [Sample 1](https://platform.sublime.security/messages/9d4702f25c5414ae9da2cedbd881d14460e3041458db8a99a1f3312dc0015697)

## Associated hunts

- [Hunt 1](https://platform.sublime.security/hunts/1e7ad6c9-05c5-4eee-a3fd-99d06a520724)

### CODE REVIEW

**Comment by zoomequipd** [comment]
looking good, ready for review, ASR rule.

File: detection-rules/abuse_dropbox_unsolicited_reply-to.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: "Service Abuse: DropBox Share From an Unsolicited Reply-To Address"
```
Comment (aidenmitchell) [inline detection-rules/abuse_dropbox_unsolicited_reply-to.yml:1]:
```suggestion
name: "Service Abuse: Dropbox Share From an Unsolicited Reply-To Address"
```

**Pull request merged**